lib/receivex/parsers.ex

```diff
@@ -0,0 +1,38 @@
+defmodule Receivex.Parsers do
+  @moduledoc false
+  def to_datetime(timestamp) when is_integer(timestamp) do
+    case DateTime.from_unix(timestamp) do
+      {:ok, timestamp} ->
+        timestamp
+
+      _ ->
+        nil
+    end
+  end
+
+  def to_datetime(timestamp) when is_float(timestamp) do
```

Comment:
do any of the api's return float timestamps? 

I feel we can be more assertive here and raise when it doesn't conform to a timestamp, and stick to either strings or integers.